### DISCUSSION TITLE
Busca Fonetica BR para SOLR

### DISCUSSION THREAD

**Discussion by marcusvcribeiro** [frontendbr/forum#1252] [Discussão]
Alguem daqui ja mexeu com Busca Fonetica Brasileira? 
Estou desenvolvendo alguns plugins e tive a necessidade de criar uma busca fonetica brasileira, entretanto só tem em PHP, e o software que estou mexendo é o SOLR, que é feito em java.

Estou com inumeras duvidas de como implementar a busca fonetica brasileira.

**Comment by tunnes** [comment]
Não conheço do SOLR mas ele tbm usa o `apache  lucene` como o ES né? Já á avaliou usar a `full text search` do postgre mesmo? Achei bem mais prático, faz um tempo que migramos do ES para a busca textual do postgre mesmo, uma olhada na [doc](https://www.postgresql.org/docs/9.5/static/textsearch.html), também te indico esse [artigo](https://www.infoq.com/br/articles/postgresql-fts) bacana falando sobre.

**Comment by tunnes** [comment]
@LFeh Nesse caso acho que ta mais pras tags de `Performance` ou `Ferramenta`, tipo no caso disso ai é sobre indexação de termos pra melhorar a velocidade da consulta do banco de dados como um `like` só que bemmm melhor. Também lembrei o @marcusvcribeiro que é uma boa ver com o pessoal do [backend-br](https://github.com/backend-br), eles podem melhor lhe orientar sobre esse tópico.

**Comment by felipefialho** [comment]
@tunnes Sim, viajei hehe

**Comment by marcusvcribeiro** [comment]
Oi galera, então, era o desenvolvimento do plugin mesmo, consegui desenvolver o plugin de Busca Fonetica Brasileira, Depois irei jogar no github, obrigado galera :)

**Comment by Maxwbh** [comment]
Bom dia, 
Eu desenvolvi a Busca Fonética em PL/SQL,(ORACLE)  esta super simples para ler o código é tranquilo para transcreve para postgress .
https://github.com/Maxwbh/FoneticaBR
